StripePaymentSheet/StripePaymentSheetTests/PaymentSheet/PaymentSheetConfigurationTests.swift

```diff
@@ -60,4 +60,28 @@ class PaymentSheetConfigurationTests: XCTestCase {
         configuration.billingDetailsCollectionConfiguration.phone = .always
         XCTAssert(configuration.isUsingBillingAddressCollection())
     }
+
+    func testSTPPaymentMethodBillingDetailsToPaymentSheetBililngDetails() {
+        var billingDetails = STPPaymentMethodBillingDetails()
+        billingDetails.name = "Jane Doe"
+        billingDetails.email = "janedoe@test.com"
+        billingDetails.phone = "+18885551234"
+        billingDetails.address = STPPaymentMethodAddress()
+        billingDetails.address?.line1 = "123 Main Street"
+        billingDetails.address?.line2 = ""
+        billingDetails.address?.city = "San Francisco"
+        billingDetails.address?.state = "California"
+        billingDetails.address?.country = "US"
+
+        let psBillingDetails = billingDetails.toPaymentSheetBillingDetails()
```

Comment:
```suggestion
        let psBillingDetails: PaymentSheet.BillingDetails = billingDetails.toPaymentSheetBillingDetails()
```

supernit, just to make it a little clearer what's going on here